web/src/ui_report.ts

```diff
@@ -110,3 +122,13 @@ export function loading(
 
     $status_box.addClass("show");
 }
+
+export function initialize(): void {
+    $("#missing-unreads-jump-to-first-unread").on("click", function (this: HTMLElement, e) {
+        if (jump_to_first_unread_callback) {
+            jump_to_first_unread_callback(this);
+        }
+        e.preventDefault();
+        e.stopPropagation();
+    });
+}
```

Comment:
It's an abstraction violation to add this to the common `ui_report` module; you can just have the `message_view` module set up the handler itself listening on body during initialization, without any independency with this module.

Or just create the handler just after calling `ui_report.show`.

Reply:
Added the event handler is message_view.show